app/forms/idv/ssn_format_form.rb

```diff
@@ -37,18 +35,5 @@ def submit(params)
     def updating_ssn?
       @updating_ssn
     end
-
-    private
-
-    def consume_params(params)
```

Comment:
it seems like this was a construct developed for forms that are "consuming" a number of different params; while i might argue about whether or not it's that useful generally, it certainly seems like overkill for a form that is only using one param.